cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -208,20 +212,36 @@ def configure_ua(token=None, enable=None, config=None):
             error=str(e)
         )
         util.logexc(LOG, msg)
-        raise RuntimeError(msg) from e
+
+        if not re.search("This machine is already attached to", str(e)):
+            raise RuntimeError(msg) from e
+
     enable_errors = []
+    enable_warnings = []
     for service in enable:
         try:
             cmd = ["ua", "enable", "--assume-yes", service]
             subp.subp(cmd, capture=True)
         except subp.ProcessExecutionError as e:
-            enable_errors.append((service, e))
+            if re.search("is already enabled.", str(e)):
+                enable_warnings.append((service, e))
+            else:
+                enable_errors.append((service, e))
+
+    if enable_warnings:
+        for service, warning in enable_warnings:
+            msg = 'Service "{service}" already enabled:\n{warning}'.format(
+                service=service, warning=str(warning)
+            )
+            util.logexc(LOG, msg)
```

Comment:
Shouldn't be this logged as a warning?

```suggestion
            LOG.warning(msg)
```